frameworks/cassandra/universe/marathon.json.mustache

```diff
@@ -74,6 +74,7 @@
     "VIRTUAL_NETWORK_PLUGIN_LABELS": "{{service.virtual_network_plugin_labels}}",
     {{/service.virtual_network_enabled}}
     {{#service.region}}
+    "ALLOW_REGION_AWARENESS": "true",
```

Comment:
Why have this envvar at all if it's hard-coded to true?  Couldn't it just be added to Cassandra's Scheduler config programmatically?

Reply:
Oh, I see what you mean. I suppose it could.

Reply:
Well, `SchedulerConfig` would have to be modified to support overrides.